applications/GeoMechanicsApplication/tests/dirichlet_release/ProjectParameters_stage1.json

```
@@ -0,0 +1,155 @@
+                            "WriteConditionsFlag":   "WriteElementsOnly",
+                            "GiDPostMode":           "GiD_PostAscii",
+                            "MultiFileFlag":         "SingleFile"
+                        },
+                        "file_label":          "step",
+                        "output_control_type": "step",
+                        "output_interval":    1,
+                        "body_output":         true,
+                        "node_output":         false,
+                        "skin_output":         false,
+                        "plane_output":        [],
+                        "nodal_results":       ["WATER_PRESSURE","DISPLACEMENT","TOTAL_DISPLACEMENT"],
+			            "gauss_point_results": ["CAUCHY_STRESS_TENSOR", "GREEN_LAGRANGE_STRAIN_TENSOR"]
+                    }
+                }
+            }
+        }]
+    },
+    "processes": {
+        "constraints_process_list": [{
+        "python_module": "apply_scalar_constraint_table_process",
+        "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name":  "ApplyScalarConstraintTableProcess",
+        "Parameters":    {
+            "model_part_name": "PorousDomain.Bottom",
+            "variable_name":   "WATER_PRESSURE",
+            "active":          true,
+            "is_fixed":        true,
+            "value":           -100.0,
+            "table":           0
+        }
+    },{
+        "python_module": "apply_scalar_constraint_table_process",
+        "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name":  "ApplyScalarConstraintTableProcess",
+        "Parameters":    {
+            "model_part_name": "PorousDomain.Top",
+            "variable_name":   "WATER_PRESSURE",
+            "active":          true,
+            "is_fixed":        true,
+            "value":           -100.0,
+            "table":           0
+        }
+    },{
+        "python_module": "apply_vector_constraint_table_process",
+        "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name":  "ApplyVectorConstraintTableProcess",
+        "Parameters":    {
+            "model_part_name": "PorousDomain.Bottom",
+            "variable_name":   "DISPLACEMENT",
+            "active":          [true,true,true],
+            "is_fixed":        [false,true,true],
+            "value":           [0.0,0.0,0.0],
+            "table":           [0,0,0]
+        }
+    },{
+        "python_module": "apply_vector_constraint_table_process",
+        "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+        "process_name":  "ApplyVectorConstraintTableProcess",
+        "Parameters":    {
+            "model_part_name": "PorousDomain.Left",
+            "variable_name":   "DISPLACEMENT",
+            "active":          [true,false,true],
+            "is_fixed":        [true,false,true],
```

For consistency with the previous process, I would change some of the flags:
```suggestion
            "active":          [true,true,false],
            "is_fixed":        [true,false,false],
```

---

Done